src/table/Exposure.cc

```diff
@@ -417,7 +384,7 @@ PTR(ExposureTable) ExposureTable::make(Schema const & schema) {
             "Schema for Exposure must contain at least the keys defined by makeMinimalSchema()."
         );
     }
-    return std::make_shared<ExposureTableImpl>(schema);
+    return std::shared_ptr<ExposureTable>(new ExposureTable(schema));
```

Comment:
The `ExposureTable` constructor is protected, and hence `make_shared` can't access it (and I don't know how to friend it).